src/features/contributor-activity/components/GitHubTokenInput.tsx

```
@@ -0,0 +1,71 @@
+  }, [token, onTokenChange]);
+
+  const handleClear = useCallback(() => {
+    setToken('');
+    localStorage.removeItem(TOKEN_KEY);
+    onTokenChange('');
+  }, [onTokenChange]);
+
+  return (
+    <div className="github-token-input">
+      <div className="token-field">
+        <label>GitHub Token (optional - for higher rate limits):</label>
```

Biome (2.4.4)

[error] A form label must be associated with an input.

(lint/a11y/noLabelWithoutControl)